src/psycopmlutils/timeseriesflattener/resolve_multiple_functions.py

```diff
@@ -53,3 +54,10 @@ def get_sum_in_group(grouped_df: DataFrame) -> DataFrame:
 @resolve_fns.register("count")
 def get_count_in_group(grouped_df: DataFrame) -> DataFrame:
     return grouped_df.count()
+
+
+@resolve_fns.register("slope")
+def get_slope_in_group(grouped_df: DataFrame) -> DataFrame:
```

Comment:
Maybe rename to be more descriptive, and consider adding a docstring. E.g. "get change in value per day" or similar. For "slope" it isn't explicit what the X-axis is :-)

Didn't do it for the above because they are _extremely_ simple.